scripts/run_tests.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script runs all the tests, in this order:
+- Frontend Karma unit tests
+- Backend Python tests
+- End-to-end Protractor tests
+
+If any of these tests result in errors, this script will terminate.
+
+Note: The test scripts are arranged in increasing order of time taken. This
+enables a broken build to be detected as quickly as possible.
+"""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import python_utils
+
+from . import run_backend_tests
+from . import run_e2e_tests
+from . import run_frontend_tests
+from . import setup
+from . import setup_gae
+
+
+def main():
+    """Run all the tests."""
+    setup.main()
+    setup_gae.main()
```

Comment:
Explicitly pass empty args

Reply:
Setup and setup_gae accepts no arguments and hence its main accepts no argv

Reply:
See below.